packages/crlfuzz/PKGBUILD

```
@@ -0,0 +1,38 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=crlfuzz
+pkgver=v1.4.1.r4.gc960f98
+pkgrel=1
+pkgdesc='A fast tool to scan CRLF vulnerability written in Go'
+url='https://github.com/dwisiswant0/crlfuzz'
+groups=('blackarch' 'blackarch-webapp')
+arch=('any')
```

Golang packages produce binaries, add specific architectures.
```suggestion
arch=('x86_64' 'aarch64')
```